test/Geometry/GeometryTestAlg.cxx

```diff
@@ -2338,6 +2344,35 @@ namespace geo{
   }
 
   //......................................................................
+  unsigned int GeometryTestAlg::planeAlignmentTo
+    (geo::PlaneGeo const& plane, geo::Vector_t const& wireDir) const
+  {
+    /*
+     * Return value:
+     *  * 0: not aligned with anything
+     *  * 1: aligned to width
+     *  * 2: aligned to depth
+     *  * 3: aligned to wires
+     */
+    
+    auto const isOrthogonalTo = [&wireDir](geo::Vector_t const& other)
+      { return std::abs(geo::vect::dot(wireDir, other)) < 1.0e-3; };
+    
+    // we dislike well-aligned wires; and we assume that if a wire is
+    // orthogonal to one of the main plane directions (dot product zero)
+    // it is parallel to the other one
+    if (isOrthogonalTo(plane.WidthDir<geo::Vector_t>()))
+      return 1;
+    if (isOrthogonalTo(plane.DepthDir<geo::Vector_t>()))
+      return 2;
+    if (isOrthogonalTo(plane.GetIncreasingWireDirection<geo::Vector_t>()))
+      return 3;
```

Comment:
Are these deliberately disconnected if statements or could `else if` be used here?

Reply:
`else if` could have been used. I did not because each branch `return`s.
Incidentally, the last check may be overlapping one of the other two if the wire is aligned to the frame of the plane.